stabilizer/src/main/java/com/hazelcast/stabilizer/agent/workerjvm/WorkerJvmLauncher.java

```diff
@@ -124,10 +129,27 @@ private WorkerJvm startWorkerJvm(String mode) throws IOException {
         new WorkerJvmProcessOutputGobbler(process.getInputStream(), new FileOutputStream(logFile)).start();
         workerJvm.process = process;
         workerJvm.mode = WorkerJvm.Mode.valueOf(mode.toUpperCase());
+        uploadResourcesToWorker(workerId);
```

Comment:
I would call this copyResourcesToWorkerId, since the uploading afaik already is done.